M2/Macaulay2/m2/expressions.m2

```diff
@@ -533,21 +533,21 @@ toString'(Function, SparseMonomialVectorExpression) := (fmt,v) -> toString (
 MatrixExpression = new HeaderType of Expression
 MatrixExpression.synonym = "matrix expression"
 matrixOpts := x -> ( -- helper function
-    opts := hashTable{CompactMatrix=>compactMatrixForm,BlockMatrix=>null,Degrees=>null};
+    opts := hashTable{CompactMatrix=>compactMatrixForm,BlockMatrix=>null,Degrees=>null,mutable=>false};
```

Comment:
How about using `MutableMatrix` instead?  That way we can adhere to the convention that optional arguments have capitalized names.

Reply:
oops, good point